src/style.css

```diff
@@ -0,0 +1,112 @@
+* {
+  font-size: 16px;
+  box-sizing: border-box;
+}
+
+body {
+  background-color: bisque;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+h1 {
+  width: 100%;
+  font-size: 16px;
```

Comment:
In order to follow DRY practices, you do not need to mention `font-size: 16px;` as it is already mentioned in [line 2](https://github.com/asdt560/Review-repo/blob/5c9598f41427b83c02de69d2e0bc770fc8c41913/src/style.css#L2) with the asterisk, therefore, setting it up for the whole project